lib/sentry/plug_context.ex

```diff
@@ -103,16 +126,7 @@ defmodule Sentry.PlugContext do
   @scrubbed_value "*********"
```

Comment:
on most SDKs, we use `[Filtered]` to signify scrubbing just in case you want to make it consistent.

Reply:
Awesome, yes I can do that in another PR 👍